src/library/scala/io/Position.scala

```diff
@@ -36,7 +36,6 @@ package io
  *  encode(line1,column1) <= encode(line2,column2)
  *  }}}
  */
-@deprecated("this class will be removed", "2.10.0")
```

Comment:
Or maybe not.

Reply:
it is `private[scala]` and disentangling it from `io.Source` looks like work so I don't see the point of the deprecation